README.md

```diff
@@ -1,4 +1,15 @@
-# gamesproject
+# The Space Invaders Project
 
-This is a work in progress on my final project to **CS50’s Web Programming with Python and JavaScript**
-this README will be updated as i go along..
+[![Run-Tests-Action Actions Status](https://github.com/AvshRLev/gamesproject/workflows/Run-Tests-Action/badge.svg)](https://github.com/AvshRLev/gamesproject/actions)
+
+[![Lint Code Base Action Actions Status](https://github.com/AvshRLev/gamesproject/workflows/Lint%20Code%20Base%20Action/badge.svg)](https://github.com/AvshRLev/gamesproject/actions)
+
+This is a project created end to end by me for learning and showcasing my abilities.
+To view the project online follow [this link](https://space-invaders.avshi.net/, "Link to my project online")
+
+## Table of contents
```

Comment:
imporve this table of contents to match this list:

Description
Edit
readme should contain:
- ci/cd tags: build, passing test, coverage, lint, etc..
- short & impressive project description (summery that contains lots of buzzwords) + link to project online.
- project architecture (use draw.io to draw it) - server, client, db
- deployment architecture (k8s) on aws
- deployment pipeline
- best practices learned (take also from the quizes in avshi wacky program) with references to files, or to comments on PRs
- inspirations - links to articles, vidoes, style guides, blog posts etc..
- link to test report
- development - instructions of how to run it locally: install python, venv, node, run webpack, or alternativly how to build and run the docker